### ISSUE TITLE
Verify RNW 0.62 Desktop DLL Does not Have vcruntime140_1.dll Dependency

### ISSUE THREAD

**Issue by NickGerleman** [microsoft/react-native-windows#4900]
VC tools 14.25 regressed FH4 behavior leading to calls to CxxFrameHandler4 where there previously weren't any. This breaks anyone not building with V142, likely regardless of if they're using the ABI safe API. We should check what we're building with. See https://github.com/microsoft/v8-jsi/pull/13 for reference.

**Comment by NickGerleman** [comment]
May also want to check the experimental MSRN Nuget. Not sure the Stellar story.

**Comment by chrisglein** [comment]
So it sounds like the change to CxxFrameHandler4 worked but had side effects? Should we revert and just go back to a custom Office forked solution?

**Comment by NickGerleman** [comment]
> So it sounds like the change to CxxFrameHandler4 worked but had side effects? Should we revert and just go back to a custom Office forked solution?

My understanding is that the FH4 change will sometimes noop depending on certain toolchain. No side effects.

**Comment by NickGerleman** [comment]
Previous flags broke with an MSVC update, so binaries have not been v141 compatible for some time. The main V141 consumer (Office) is unblocked to v142 though, so this is no longer needed. A separate issue still tracks removing flags.
